x/pairing/types/message_freeze_test.go

```diff
@@ -0,0 +1,40 @@
+package types
+
+import (
+	"testing"
+
+	sdkerrors "github.com/cosmos/cosmos-sdk/types/errors"
+	"github.com/lavanet/lava/testutil/sample"
+	"github.com/stretchr/testify/require"
+)
+
```

Comment:
Should also test:
- freeze a frozen provider should fail
- freeze an unstaking provider should fail
- frozen provider gets properly paid for (recent) past services despite being frozen

Reply:
done :)